yotta/build.py

```diff
@@ -27,6 +33,13 @@ def addOptions(parser, add_build_targets=True):
         action='store_true', default=False,
         help='Only generate CMakeLists, don\'t run CMake or build'
     )
+    parser.add_argument('-x', '--export', dest='export',
+        default=False, const=True, nargs='?',
+        help=(
+            'Export mode. If flag is set, generates builds without reference to Yotta.'
```

Comment:
`yotta` not `Yotta`, or perhaps remove this comment? e.g. `Export mode. If flag is set, generates a buildable folder structure in the specified folder.